overlord/snapstate/handlers.go

```diff
@@ -1821,14 +1827,77 @@ func (m *SnapManager) stopSnapServices(t *state.Task, _ *tomb.Tomb) error {
 	st.Lock()
```

Comment:
I think the comment above queryDisabledServices could use to be rewritten now, given that the we no longer do things in doLinkSnap, etc